app/poros/event.rb

```diff
@@ -4,17 +4,17 @@ class Event
               :gym_membership_id,
               :date_time,
               :activity,
-              :invitee_name,
-              :role
+              :friend_name,
+              :friend_role
```

Comment:
Thank you for updating these - apologies to leave this open yesterday in my last PR to realign FE to BE endpoint updates

Reply:
You got it!  Great communication on that.